_posts/developers/apps/inrupt-tutorial/2020-11-06-00_getting-started.md

```diff
@@ -112,398 +108,390 @@ Node.js installation.
-        import { VCARD } from "@inrupt/vocab-common-rdf";
-        
-        // If your Pod is *not* on `solidcommunity.net`, change this to your identity provider.
-        const IDENTITY_PROVIDER = "https://solidcommunity.net";
-        document.getElementById("identity_provider").innerHTML = `[<a target="_blank" href="${IDENTITY_PROVIDER}">${IDENTITY_PROVIDER}</a>]`;
-      
-        const NOT_ENTERED_WEBID = "...not logged in yet - but enter any WebID to read from its profile...";
-        
-        const session = new Session();
-        
-        const buttonLogin = document.querySelector("#btnLogin");
-        const buttonWrite = document.querySelector("#btnWrite");
-        const buttonRead = document.querySelector("#btnRead");
-      
-        // Click button if the user hits the 'Enter' key when entering name.
-        document.querySelector("#input_name").addEventListener("keyup", event => {
-          if(event.key === "Enter") {
-            buttonWrite.click();
-            event.preventDefault();
-          }
-        });
-        
-        // Click button if the user hits the 'Enter' key when entering WebID.
-        document.querySelector("#webID").addEventListener("keyup", event => {
-          if(event.key === "Enter") {
-            buttonRead.click();
-            event.preventDefault();
-          }
+1. Create an `index.js` file with the following content:   
+
+    > **NOTE:** If your Pod is not on `https://solidcommunity.net`, modify the
+      value of the `SOLID_IDENTITY_PROVIDER` variable accordingly.
+  
+    ```javascript
+    import {
+      getSolidDataset,
+      getThing,
+      setThing,
+      getStringNoLocale,
+      setStringNoLocale,
+      saveSolidDatasetAt
+    } from "@inrupt/solid-client";
+    
+    import { Session } from "@inrupt/solid-client-authn-browser";
+    
+    import { VCARD } from "@inrupt/vocab-common-rdf";
+    
+    // If your Pod is *not* on `solidcommunity.net`, change this to your Solid identity provider.
+    const SOLID_IDENTITY_PROVIDER = "https://solidcommunity.net";
+    document.getElementById("solid_identity_provider").innerHTML = `[<a target="_blank" href="${SOLID_IDENTITY_PROVIDER}">${SOLID_IDENTITY_PROVIDER}</a>]`;
+  
+    const NOT_ENTERED_WEBID = "...not logged in yet - but enter any WebID to read from its profile...";
+    
+    const session = new Session();
+    
+    const buttonLogin = document.querySelector("#btnLogin");
+    const buttonWrite = document.querySelector("#btnWrite");
+    const buttonRead = document.querySelector("#btnRead");
+  
+    // Click button if the user hits the 'Enter' key when entering name.
+    document.querySelector("#input_name").addEventListener("keyup", event => {
+      if(event.key === "Enter") {
+        buttonWrite.click();
+        event.preventDefault();
+      }
+    });
+    
+    // Click button if the user hits the 'Enter' key when entering WebID.
+    document.querySelector("#webID").addEventListener("keyup", event => {
```

Comment:
While we're at it, might use the right tool for the job as well:

```suggestion
    const buttonLogin = document.getElementById("btnLogin");
    const buttonWrite = document.getElementById("btnWrite");
    const buttonRead = document.getElementById("btnRead");
  
    // Click button if the user hits the 'Enter' key when entering name.
    document.getElementById("input_name").addEventListener("keyup", event => {
      if(event.key === "Enter") {
        buttonWrite.click();
        event.preventDefault();
      }
    });
    
    // Click button if the user hits the 'Enter' key when entering WebID.
    document.getElementById("webID").addEventListener("keyup", event => {
```

Although it looks like this manually captures "Enter" presses in input fields to submit the form? That's not great - if buttons are of `type="submit"` and form submissions are handled in the form's submit events, then the browser will handle this automatically and reliably for input fields in the same form.

Anyway, that's a bit too much for this PR, but as a reminder to myself to fix this up afterwards.